static-assets/scripts/host.js

```diff
@@ -310,7 +310,7 @@
         studioPath = CStudioAuthoring.ComponentsPanel.getPreviewPagePath(message.url);
       selectStudioContent(site, studioPath);
 
-      CStudioAuthoringContext.previewCurrentPath = message.url;
+      setHash({ page: message.url, site });
```

Comment:
`CStudioAuthoringContext.previewCurrentPath = message.url;` is done in `setHash`